PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-kong.yml

```diff
@@ -0,0 +1,57 @@
+image:
+  repository: quay.io/fiware/kong
+  tag: "0.3.0-PRE-10"
```

Comment:
We should use a stable version here, e.g. 0.2.0

Reply:
ok